can/interfaces/ics_neovi/neovi_bus.py

```diff
@@ -314,7 +329,11 @@ def send(self, msg, timeout=None):
         message.StatusBitField = flag0
         message.StatusBitField2 = 0
         message.StatusBitField3 = flag3
-        message.NetworkID = self.network
+        if msg.channel is not None:
+            message.NetworkID = msg.channel
+        else:
+            # defaults to the first channel in channels
```

Comment:
This could be surprising! We should at least document this clearly, or maybe even raise an Exception? Another option would be to send it to all, but I guess that is unexpected as well?